packages/shorebird_cli/lib/src/commands/release/ios_releaser.dart

```diff
@@ -98,13 +103,20 @@ class IosReleaser extends Releaser {
     final flutterVersionString = await shorebirdFlutter.getVersionAndRevision();
     final buildProgress =
         logger.progress('Building ipa with Flutter $flutterVersionString');
+    final publicKeyFile = argResults.file('public-key-path');
+
+    final base64PublicKey = publicKeyFile != null
+        ? base64Encode(publicKeyFile.readAsBytesSync())
+        : null;
```

Comment:
We should probably move this up to the Releaser/ReleaseCommand level, as this will be applicable for all platforms

Reply:
Agreed! I moved it to the `ArgResults` extension, that way we can share this logic in other commands too? Not sure if an extension is the most appropriate way, but I feel it kinda worked well, lmk what you think